redbeams/src/main/java/com/sequenceiq/redbeams/aspect/ControllerLogContextAspects.java

```diff
@@ -0,0 +1,55 @@
+package com.sequenceiq.redbeams.aspect;
+
+import java.util.Arrays;
+
+import javax.inject.Inject;
+
+import org.aspectj.lang.JoinPoint;
+import org.aspectj.lang.annotation.Aspect;
+import org.aspectj.lang.annotation.Before;
+import org.aspectj.lang.annotation.Pointcut;
+import org.aspectj.lang.reflect.CodeSignature;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+
+import com.sequenceiq.cloudbreak.logger.LogContextService;
+
+@Component
+@Aspect
+public class ControllerLogContextAspects {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(ControllerLogContextAspects.class);
+
+    @Inject
+    private LogContextService logContextService;
+
+    @Pointcut("execution(public * com.sequenceiq.redbeams.controller..*(..))")
+    public void interceptControllerMethodCalls() {
+    }
+
+    @Before("com.sequenceiq.redbeams.aspect.ControllerLogContextAspects.interceptControllerMethodCalls()")
```

Comment:
What's the point of interceptControllerMethodCalls here? I'm guessing just syntactic nicety, but it's not actually doing anything useful here AFAICT.

Reply:
It's how AspectJ defines pointcuts. You can see the method call stated in the @Before annotation right afterwards. If you take a look at SecretAspects, you can see several more examples. :D

Reply:
I have a feeling this isn't necessary, but I don't feel that strongly about this. Check out the PipelineJobProfiler in Director. I don't think the empty method with a Pointcut is necessary.